relentless/simulate/analyze.py

```diff
@@ -18,3 +18,29 @@ class EnsembleAverage(simulate.GenericAnalysisOperation):
 
     def __init__(self, check_thermo_every, check_rdf_every, rdf_dr):
         super().__init__(check_thermo_every, check_rdf_every, rdf_dr)
+
+
+class WriteTrajectory(simulate.GenericAnalysisOperation):
+    """Writes a .gsd file.
+
+    Parameters
+    ----------
+    filename : str
+        Name of the .gsd file to be written.
+    every : int
+        Interval of time steps at which to write a snapshot of the simulation.
+    velocity : bool
+        Log particle velocity.
+    image : bool
+        Log particle image.
+    typeid : bool
```

Comment:
We call these "types" in relentless. However, `type` is reserved... maybe use plurals like "velocities", "types", etc.?